app/controllers/revisions_controller.rb

```diff
@@ -40,6 +40,44 @@ def show
             )
   end
 
+  def trash
+    # Get all versions whose event is destroy.
+    @revisions = PaperTrail::Version.where(event: 'destroy').order(created_at: :desc)
```

Comment:
Looks like it. Shouldn't be too hard to add the scope, but do you want to do this by monkey-patching `PaperTrail::Version` with a method called `set_project_scope`, or do it some other way? Monkey-patching is generally frowned upon but it might be the least bad option here.

(Note that we can't just call `PaperTrail::Version.default_scope` directly from the controller since `default_scope` is a protected method)

And do we need to modify anything in this PR for this, or can it wait until we merge into Pro?